fedimint-server/src/net/api.rs

```diff
@@ -296,6 +310,73 @@ impl ConsensusApi {
             .await
             .map_err(|_| ApiError::server_error("Unable send event".to_string()))
     }
+
+    pub async fn build_full_server_status(&self) -> ApiResult<FullStatus> {
```

Comment:
That "full" is meh and it keeps repeating :)

Reply:
`build` suggest some kind of a builder. While this seems to just "return" (the status).